src/test/mir-opt/const_prop/control_flow_simplification.hello.ConstProp.diff.all

```diff
@@ -41,19 +41,19 @@
       }
   
       bb2: {
-          StorageLive(_2);                 // scope 0 at $SRC_DIR/std/src/macros.rs:LL:COL
-          const std::rt::begin_panic::<&str>(const "explicit panic"); // scope 0 at $SRC_DIR/std/src/macros.rs:LL:COL
+          StorageLive(_2);                 // scope 0 at $LIB_DIRrary/std/src/macros.rs:13:23: 13:52
+          const std::rt::begin_panic::<&str>(const "explicit panic"); // scope 0 at $LIB_DIRrary/std/src/macros.rs:13:23: 13:52
                                            // ty::Const
                                            // + ty: fn(&str) -> ! {std::rt::begin_panic::<&str>}
                                            // + val: Value(Scalar(<ZST>))
                                            // mir::Constant
-                                           // + span: $SRC_DIR/std/src/macros.rs:LL:COL
+                                           // + span: $LIB_DIRrary/std/src/macros.rs:13:23: 13:46
```

Comment:
This regressed  (probably during the rebase over the latest directory restructuring)

Reply:
wow... what a bizarre diff, i'll go fix it